sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -3257,6 +3257,83 @@ def from_dict(cls, data: Dict) -> "DocumentModelSummary":
         )
 
 
+class DocumentClassifierDetails:
+    """Document classifier information. Includes the doc types that the model can classify."""
+
+    classifier_id: str
+    """Unique document classifier name."""
+    description: Optional[str]
+    """Document classifier description."""
+    created_date_time: datetime.datetime
+    """Date and time (UTC) when the document classifier was created."""
+    expiration_date_time: Optional[datetime.datetime]
+    """Date and time (UTC) when the document classifier will expire."""
+    api_version: str
+    """API version used to create this document classifier."""
+    doc_types: Dict[str, Any]
```

Comment:
```suggestion
    doc_types: Dict[str, ClassifierDocumentTypeDetails]
```
ClassifierDocumentTypeDetails returns container url with sas token